pytorch_lightning/models/trainer.py

```diff
@@ -410,19 +414,20 @@ def fit(self, model):
 
         # when using multi-node or DDP within a node start each module in a separate process
         if self.use_ddp:
-            # must copy only the meta of the exp so it survives pickle/unpickle when going to new process
+            # must copy only the meta of the exp so it survives pickle/unpickle
+            #  when going to new process
             self.experiment = self.experiment.get_meta_copy()
 
             if self.is_slurm_managing_tasks:
                 task = int(os.environ['SLURM_LOCALID'])
                 self.ddp_train(task, model)
             else:
-                msg = f"""
-                You requested {self.nb_requested_gpus} GPUs but launched {self.nb_slurm_tasks} slurm tasks. 
-                We will launch {self.nb_requested_gpus} processes for you. 
-                We recommend you let slurm manage the processes by setting: --ntasks-per-node={self.nb_requested_gpus}
-                If you're not using SLURM, ignore this message!
-                """
+                msg = """
+You requested %(nb_gpus)s GPUs but launched %(nb_tasks)s slurm tasks.
+We will launch %(nb_gpus)s processes for you.
+We recommend you let slurm manage the processes by setting: --ntasks-per-node=%(nb_gpus)s
+If you're not using SLURM, ignore this message!
+""" % {'nb_gpus': self.nb_requested_gpus, 'nb_tasks': self.nb_slurm_tasks}
```

Comment:
why make the string so complicated with dictionary mapping? why not keep it simpler to read with f strings?

Reply:
the one variable is printed in the text multiple times, so now you pass it just once...

Reply:
good point